configfile.js

```diff
@@ -475,7 +475,7 @@ cfreader.load_flat_config = function(name, type) {
             var data   = fs.readFileSync(name, "UTF-8");
             if (type === 'data') {
                 while (data.length > 0) {
-                    var match = data.match(/^([^\n]*)\n?/);
+                    var match = data.match(/^([^\r\n]*)\r?\n?/);
```

Comment:
shouldn't this change be in a separate PR, with an explanation for why it's being made?